src/ui/Templates/Template.ts

```diff
@@ -78,7 +78,7 @@ export class Template implements ITemplateProperties {
 
```

Comment:
Future improvement would be to tell the user which result they are looking at. e.g. "Result 2 out of 10"
I created a separate jira for that: https://coveord.atlassian.net/browse/JSUI-2244